tutoroars/templates/oars/apps/data/assets/datasets/OpenedX_Clickhouse/xapi_events_all_parsed.yaml

```diff
@@ -0,0 +1,186 @@
+{% raw %}
+table_name: xapi_events_all_parsed
```

Comment:
@bmtcril 

> I think there may be some changes we'll need to sync up on re: ClickHouse table names based on the Ralph work, but I'll ping you on that once I get those PRs ready / merged.

Thanks! I started using your new `xapi_events_all_parsed` table -- is that the right one?

Reply:
That's the one! 🎉